Update README.md (#527)
origranot/reduced.to: README.md

[![Preview](/assets/images/preview.gif)](https://reduced.to)

  <p align="center">

>     Reduced.to is a modern web application that reduces the length of the link URL. So it's easier to remember, share and track.

Reviewer: I think this is a better explanation:

```suggestion
Reduced.to is a modern web application that shortens the length of link URLs, making it easier to remember, share, and track.
```
